dev.env

```diff
@@ -85,6 +85,7 @@ case "$MYSQL_FLAVOR" in
     ;;
   "MariaDB")
     export LD_LIBRARY_PATH=$(prepend_path $LD_LIBRARY_PATH $VT_MYSQL_ROOT/lib)
+    export CGO_CFLAGS="$CGO_CFLAGS -I$VT_MYSQL_ROOT/include"
```

Comment:
Ideally, we should be using pkg-config for this. But I don't remember how I got it working. I'll need to brush up.